src/lib/net/src/packets/outgoing/login_disconnect.rs

```diff
@@ -0,0 +1,14 @@
+use ferrumc_macros::{packet, NetEncode};
+use std::io::Write;
+
+#[derive(NetEncode)]
+#[packet(packet_id = 0x00)]
```

Comment:
Is this correct packet ID? It might be, I haven't checked, just seems like a coincidence.